.github/workflows/build.yml

```
@@ -30,8 +30,8 @@ jobs:
             BASE_IMAGE_NAME=${{ matrix.base_image_name }}
             BASE_IMAGE_TAG=${{ matrix.base_image_tag }}
             VCS_REF=${{ github.sha }}
-            ROS_DISTRO=''
-            ROS_APT_REPO_URL='http://packages.ros.org/ros'
+            ROS_DISTRO=""
+            ROS_APT_REPO_URL="http://packages.ros.org/ros http://packages.ros.org/ros2"
```

```suggestion
            ROS_APT_REPO_URLS="http://packages.ros.org/ros http://packages.ros.org/ros2"
```

and below probably too

---

Done.